scss/core/_utilities.scss

```diff
@@ -103,3 +103,11 @@ $indexes: map-merge(
     z-index: $size !important;
   }
 }
+
+@each $level in $levels {
+  @each $side in $sides {
+    .box-shadow-#{$side}-#{$level} {
```

Comment:
Does the documentation say that these CSS class names are available to use, in addition to the SCSS mixin and variables?

Reply:
Yes, added